tensilelite/Tensile/Toolchain/Assembly.py

```diff
@@ -136,7 +136,7 @@ def compress(self, srcPath: str, destPath: str, gfx: str):
             "--compress",
             "--type=o",
             "--bundle-align=4096",
-            f"--targets=host-x86_64-unknown-linux,hipv4-amdgcn-amd-amdhsa--{gfx}",
+            f"--targets=host-x86_64-unknown-linux-gnu,hipv4-amdgcn-amd-amdhsa-unknown--{gfx}",
```

Comment:
For the AMD triple, you can either use `amdgcn-amd-amdhsa-unknown-{gfx}` or `amdgcn-amd-amdhsa--{gfx}`. Having two `-` after environment is wrong.